.github/workflows/main.yml

```diff
@@ -0,0 +1,40 @@
+name: .NET Core Test and Publish
+
+on: 
+  push:
+    branches: master
+  pull_request:
+    
+env:
+  DOTNET_VERSION: 2.1
+
+jobs:
+
+  Test:
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest]
+    runs-on: ${{ matrix.os }}
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v1
+    - name: Setup .NET Core
+      uses: actions/setup-dotnet@v1
+      with:
+        dotnet-version: ${{ env.DOTNET_VERSION }}
+    - name: Check format
+      run: |
+        dotnet tool install --version 3.2.111002 --tool-path ./ dotnet-format --add-source https://dotnet.myget.org/F/format/api/v3/index.json
+        # ./dotnet-format --check --dry-run -v diagnostic
+    - name: Test
+      if: runner.os == 'Windows'
+      run: |
+        forfiles /p tests /m *.csproj /s /c "cmd /c dotnet add @PATH package coverlet.msbuild"
```

Comment:
remove /p tests; there is no tests folder under current directory.

Reply:
@cloud8little sorry i didn't read your review, could you fix it in other pr?